Subsystem/CageCatcher

CoderLiam926 opened the pull request:
(Cage Catcher and Cage Climber)

CoderLiam926 committed:
I added stuffing

not a lot though

kjrokos commented on src/main/java/frc/robot/subsystems/CageSubsystem.java:
```diff
@@ -1,10 +1,14 @@
 
 public class CageSubsystem extends SubsystemBase {
-
+SparkMax cageGrabber = new SparkMax(13, MotorType.kBrushed);
+SparkMax cageClimber = new SparkMax(13, MotorType.kBrushed);
```
Make sure these are different IDs, and add to constants file

CoderLiam926 replied:
Henry did it!

CoderLiam926 committed:
Merge branch 'main' into Subsystem/Cage

Henyahyah committed:
Cage and Climb

I copied the same thing we did with the Coral Scorer toggling thing over to the Cage for climbing.

CoderLiam926 committed:
Merge branch 'main' into Subsystem/CoralScoringDevice

Henyahyah committed:
stash work merging

Henyahyah committed:
readding code

Henyahyah committed:
Cage Grabber movement and controlsl

Pushing the down arrow on the controller should move the bar down to hold the cage in place. The up arrow should raise it back to its neutral position.

Co-Authored-By: RedArrowRobotics <79018739+RedArrowRobotics@users.noreply.github.com>

CoderLiam926 committed:
Lots of Bug Fixes for ControlInputs, RobotContainer, and CageSubsystem

I changed some inputs, deleted a repeat of code, and changed a command type to the correct type.
